common/model-views.cpp

```diff
@@ -1667,48 +1667,8 @@ namespace rs2
 
     bool subdevice_model::is_selected_combination_supported()
     {
-        std::vector<stream_profile> results;
-
-        for (auto&& f : formats)
-        {
-            auto stream = f.first;
-            if (stream_enabled[stream])
-            {
-                auto fps = 0;
-                if (show_single_fps_list)
-                    fps = shared_fps_values[ui.selected_shared_fps_id];
-                else
-                    fps = fps_values_per_stream[stream][ui.selected_fps_id[stream]];
-
-                auto format = format_values[stream][ui.selected_format_id[stream]];
-
-                for (auto&& p : profiles)
-                {
-                    if (auto vid_prof = p.as<video_stream_profile>())
-                    {
-                        if (res_values.size() > 0)
-                        {
-                            auto width = res_values[ui.selected_res_id].first;
-                            auto height = res_values[ui.selected_res_id].second;
-
-                            if (vid_prof.width() == width &&
-                                vid_prof.height() == height &&
-                                p.unique_id() == stream &&
-                                p.fps() == fps &&
-                                p.format() == format)
-                                results.push_back(p);
-                        }
-                    }
-                    else
-                    {
-                        if (p.fps() == fps &&
-                            p.unique_id() == stream &&
-                            p.format() == format)
-                            results.push_back(p);
-                    }
-                }
-            }
-        }
+        bool throw_exception_flag = false;
```

Comment:
Pls rename the var to `combination_supported` for clarity

Reply:
let's talk about this